### DISCUSSION TITLE
The application Eth Gas Station is retired

### DISCUSSION THREAD

**Discussion by brunacunha** [Cyfrin/foundry-full-course-cu#1297] [Q&A]
Hello, I would like to point out that the website Eth Gas Station is retired.

Here it is being linked on the course:

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/21175243/27aa74ca-4460-4db9-91f1-8d8fa4cf92ba)

Here's the website (https://ethgasstation.info/):

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/21175243/102a7359-9ff9-4fe6-934e-4350f59b8c4f)

Can we have an update on the course Readme?

**Comment by xyz899** [comment]
Hey, 
I use QuickNode for Real-time tracking of the Gas Price here it is :

https://www.quicknode.com/gas-tracker/ethereum
